src/test/java/uk/gov/justice/laa/crime/cfecrime/utils/RequestTestUtil.java

```diff
@@ -27,4 +40,61 @@ public static void setSectionPassportBenefit(CfeCrimeRequest request, boolean va
         sectionPassportedBenefit.withPassportedBenefit(Boolean.valueOf(value));
         request.withSectionPassportedBenefit(sectionPassportedBenefit);
     }
+
+    public static void setSectionInitMeansTest(CfeCrimeRequest cfeCrimeRequest, CaseType caseType, MagCourtOutcome magCourtOutcome)  {
+        List<DependantChild> dcList = new ArrayList<DependantChild>();
+        DependantChild dc = new DependantChild();
+        dc.setAgeRange(AgeRange.SIXTEEN_TO_EIGHTEEN);
+        dcList.add(dc);
+        Income  income = new Income();
+
+        SectionInitialMeansTest sectionInitialMeansTest = new SectionInitialMeansTest();
+        sectionInitialMeansTest.setCaseType(caseType);
+        sectionInitialMeansTest.setDependantChildren(dcList);
+        sectionInitialMeansTest.setHasPartner(false);
+        sectionInitialMeansTest.setIncome(income);
+        sectionInitialMeansTest.setMagistrateCourtOutcome(magCourtOutcome);
+
+        cfeCrimeRequest.setSectionInitialMeansTest(sectionInitialMeansTest);
+    }
+
+    public static void setSectionFullMeansTest(CfeCrimeRequest cfeCrimeRequest)  {
+        List<DependantChild> dcList = new ArrayList<DependantChild>();
+        DependantChild dc = new DependantChild();
+        dc.setAgeRange(AgeRange.SIXTEEN_TO_EIGHTEEN);
+        dcList.add(dc);
+        Income  income = new Income();
+
+        SectionFullMeansTest sectionFullMeansTest = new SectionFullMeansTest();
+
+        cfeCrimeRequest.setSectionFullMeansTest(sectionFullMeansTest);
+        AllowableOutgoings aos = new AllowableOutgoings();
+
+        SectionFullMeansTest sfmt = new SectionFullMeansTest();
+        sfmt.setAdditionalProperty("Property", "Rented");
```

Comment:
don't understand this - its not a thing as far as I know

Reply:
same as above.

Reply:
don't understand. This method should not present on the SectionFullMeansTest object - we should add additionalProperties: false to prevent this